Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -703,11 +703,10 @@ public Task ToggleDetailRow( TItem item, bool forceDetailRow = false )
         protected internal Task ToggleDetailRow( TItem item, DetailRowTriggerType detailRowTriggerType, bool forceDetailRow = false, bool skipDetailRowTriggerType = false )
         {
             var rowInfo = GetRowInfo( item );
+
             if ( rowInfo is not null )
             {
-                if ( forceDetailRow )
-                    rowInfo.ToggleDetailRow();

```

Comment:
Did you remove this?

Reply:
Yes. It seems to me that it was not needed. ` rowInfo.ToggleDetailRow();` was called in here and in `else` when `if ( DetailRowTrigger is not null )` was not satisfied. So I just left it in an `else` block.

Reply:
Maybe you missunderstood that parameter purpose, the ForceDetailRow argument is to, toggle it either way... even if DetailTrigger would evaluate to false?

Reply:
Hmm I think you're right. Going to reverse it.